sockeye/rnn_attention.py

```diff
@@ -59,6 +63,26 @@ def __init__(self,
         self.layer_normalization = layer_normalization
         self.config_coverage = config_coverage
         self.num_heads = num_heads
+        self.dtype = dtype
+
+
+def _instantiate(cls, params):
+    """
+    Helper to instantiate Attention classes from parameters. Warns in log if parameter is not supported
+    by class constructor.
+
+    :param cls: Attention class
```

Comment:
[nit] missing period at the end of param/return descriptions.

Reply:
fixed